src/benchmarkstt/output/core.py

```diff
@@ -0,0 +1,70 @@
+from benchmarkstt import output
+from benchmarkstt.schema import Schema
+from csv import writer
+import sys
+
+
+class ReStructuredText(output.Base):
+    def result(self, title, result):
+        print(title)
+        print('=' * len(title))
+        print()
+
+        if type(result) is float:
+            print("%.6f" % (result,))
+        else:
+            print(result)
+        print()
+
+
+class MarkDown(output.Base):
+    def result(self, title, result):
+        print('# %s' % (title,))
+        print()
+
+        if type(result) is float:
+            print("%.6f" % (result,))
+        else:
+            print(result)
+        print()
+
+
+class Json(output.Base):
+    def __init__(self):
+        self._line = None
+
+    def __enter__(self):
+        if self._line is not None:
+            raise ValueError("Already open")
+        print('{')
```

Comment:
It's great that we have JSON output from this tool, however I don't like how the JSON is being generated right now. I'd be more comfortable if this was done using a library instead of adding `{` characters yourself.

Reply:
It does use Schema.dumps, only the outer `{` and `}` + commas are not generated by a json.dumps, the rest is.